.github/workflows/pythonpackage.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
       - name: Run mypy
         run: |
-          mypy --disable-error-code=import-untyped .
+          mypy --disable-error-code=import-untyped --check-untyped-defs .
```

Comment:
Why add this?

Reply:
Because when I ran mypy it complained that it was not checking some things that could be relevant, and I though that it then makes sense to run mypy in a more strict way. It only affected the new code, but I think it makes sense to include this going forward.

Reply:
Here is how it looks like without it
![image](https://github.com/user-attachments/assets/d1d4050e-11c5-436e-b474-dd1ba857cc2d)